backend/routes/issue.js

```diff
@@ -242,6 +243,26 @@ function renderIssuesHTML(req, res, next) {
 	let findIssue;
 	const noClean = false;
 
+	/**
+	* Create dynamic Print report values to use in template.
+	**/
+	const reportValues = {};
+	const reportDate = moment().format("Do MMMM YYYY");
+	const currentUser = req.session.user.username;
+	reportValues.reportDate = reportDate;
+	reportValues.currentUser = currentUser;
+
+	ModelSetting.findById({ account: req.params.account, model: req.params.model }, req.params.model)
```

Comment:
This promise needs to be waited on for line #313. As it stands, it's possible that you're rendering the pug before modelName is returned. Either use async await or put this promise into an array with findIssue and do `Promise.All`